fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/Transaction.java

```diff
@@ -81,7 +85,7 @@
      * Get the date this session expires
      * @return expiration date, if one exists
      */
-    Optional<Instant> getExpires();
+    Instant getExpires();
```

Comment:
Do we want to support transactions that will not expire? or Was there any other reason for using an `Optional` here?